### PR TITLE
Features/javascript

### PR DESCRIPTION
**Pull request by DarlingDC2**
script links

### CODE REVIEW

**Review by DarlingDC2 — COMMENTED** [review]
comment

File: assets/js/script.js [hunk 1]
```diff
@@ -110,6 +110,5 @@ saveButton.addEventListener('click', handleSaveCrypto);
 
 // Call the function to load saved cryptocurrencies when the page loads
 loadSavedCryptos();
-=======
```
Comment (DarlingDC2) [inline assets/js/script.js]: ?

**Review by DarlingDC2 — COMMENTED** [review]
?

**Comment by DarlingDC2** [comment]
working?

**Review by scsc22 — APPROVED** [review]
*note that line 52 appears red, but pushing for functionality

**Pull request merged**